code/modules/power/supermatter/supermatter.dm

```diff
@@ -116,6 +116,17 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	/// don't let these to be consumed by SM
 	var/static/list/not_dustable
 
+	///Effect holder for the displacement filter to distort the SM based on its activity level
+	var/atom/movable/distortion_effect/distort
+
+/atom/movable/distortion_effect
+	plane = GRAVITY_PULSE_PLANE
+	appearance_flags = PIXEL_SCALE // no tile bound so you can see it around corners and so
```

Comment:
Not sure if this works like that when its in the vis contents of an object, if its in the vis contents it will only render when the source object is visible, so making it not pixel bound will have no effect unless it is on the turf by itself.